### ISSUE TITLE
Automatically select last wallet

### ISSUE THREAD

**Issue by marcodefelice** [BudgetControl/BudgetControl#68]

**Comment by nareshreddyetikyala** [comment]
ciao @marcodefelice i am not able to assign icon so I am going to start the story marco.

**Comment by nareshreddyetikyala** [comment]
ciao @marcodefelice 

can i store the last selected category and wallet into local storage?
or should I try another way?

**Comment by marcodefelice** [comment]
yes sure, at the moment the best way is store into a Localstorage then wi will use pinia on feature/3.0/pinia branch where I just started to work
